securityscanutils/securityscan.go

```diff
@@ -200,7 +217,16 @@ func (s *SecurityScanner) initializeRepoConfiguration(ctx context.Context, repo
 		// Create Github issues for all releases in the set
 		issuePredicate = NewLatestPatchRepositoryReleasePredicate(releasesToScan)
 	}
-	repo.githubIssueWriter = NewGithubIssueWriter(githubRepo, s.githubClient, issuePredicate)
+	if useGithubWriter {
+		repo.issueWriter = issuewriter.NewGithubIssueWriter(githubRepo, s.githubClient, issuePredicate)
+	} else if repo.Opts.OutputResultLocally {
+		repo.issueWriter, err = issuewriter.NewLocalIssueWriter(path.Join(repo.Opts.OutputDir, githubRepo.RepoName, "issue_results"))
+		if err != nil {
+			return err
+		}
+	} else {
+		repo.issueWriter = issuewriter.NewNoopWriter()
+	}
```

Comment:
The log line below this file is no longer accurate. Now that we don't always generate GithubIssueWriter this will need to be adjusted